rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbCreateAlarmNode.java

```diff
@@ -190,6 +191,19 @@ private Alarm getAlarmFromMessage(TbContext ctx, TbMsg msg) throws IOException {
         return Futures.transform(asyncUpdated, a -> new TbAlarmResult(false, true, false, a), MoreExecutors.directExecutor());
     }
 
+    private void updateStatus(Alarm existingAlarm, Alarm msgAlarm) {
+        if (!existingAlarm.getStatus().isCleared() && msgAlarm.getStatus().isCleared()) {
+            throw new RuntimeException("The 'Alarm Create' node cannot clear the alarm!");
```

Comment:
This breaks backward compatibility. Previously one was able to clear the alarm multiple times. Now this will cause the error.